shaders/lines_shader.frag

```diff
@@ -0,0 +1,7 @@
+#version 410
+
+out vec4 frag_colour;
```

Comment:
```suggestion
out vec4 frag_color;
```

Reply:
I must accept it, This still confuses me.